docs/architecture/adr-007-specialization-groups.md

```diff
@@ -0,0 +1,161 @@
+# ADR 007: Specialization Groups
+
+## Changelog
+
+- 2019 Jul 31 : Initial Conception
+
+## Context
+
+This idea was first conceived of in order to fulfill the use case of the
+creation of a decentralized Computer Emergency Response Team (CERT), Whose
+members would be elected by the Cosmos Community and would fulfill the role of
+coordinating the Cosmos community under an emergency situations. This thinking
+can be further abstracted into the conception of "blockchain specialization
+groups". 
+
+The creation of these groups are the beginning of specialization capabilities
+within a wider blockchain community which could be used to enable a certain
+level of delegated responsibilities. Examples of specialization which could be
+beneficial to a blockchain community include: code auditing, emergency response,
+code development etc. This type of community organization paves the way for
+individual stakeholders to delegate votes by issue type, if in the future
+governance proposals include a field for issue type. 
+
+
+## Decision
+
+A specialization group can be broadly broken down into
+the following functions: 
+ - Membership Admittance
+   - via governance
+   - via special appointment
```

Comment:
can you clarify this item?

Reply:
These are examples of potentially implementable situations, I'll clarify

Reply:
Added a discussion paragraph talking about this stuff and removed these bullets from under `Membership Admittance`